habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -719,9 +719,12 @@ def capture_state(self, with_articulated_agent_js=False) -> Dict[str, Any]:
         ]
         art_T = [ao.transformation for ao in self.art_objs]
         rom = self.get_rigid_object_manager()
-        static_T = [
-            rom.get_object_by_id(i).transformation for i in self._scene_obj_ids
-        ]
+        rigid_T, rigid_V = [], []
+        for i in self._scene_obj_ids:
```

Comment:
What is the reason for these changes? What do we use `capture_state` and `set_state` for?

Reply:
I will cherry pick this from main. This is needed because oracle nav sets/reset the state to calcualte where the robot should walk. Without this, whenever the robot tries to navigate to an object, the velocity of the object thrown by the human will be cleared.